src/Dapr.AspNetCore/TopicAttribute.cs

```diff
@@ -26,10 +26,29 @@ public TopicAttribute(string pubsubName, string name)
             this.PubsubName = pubsubName;
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TopicAttribute" /> class.
+        /// </summary>
+        /// <param name="pubsubName">The name of the pubsub component to use.</param>
+        /// <param name="name">The topic name.</param>
+        /// <param name="enableRawPayload">The enable/disable raw pay load flag.</param>
+        public TopicAttribute(string pubsubName, string name, bool enableRawPayload)
```

Comment:
I'd suggest skipping the optional constructor and making the property get/set instead. This is slightly more idiomatic for optional values in an attribute.

Reply:
I created a separate constructor to take care of following scenarios:
true => override global setting with true
false => override global setting with false
null => use global setting
If I use the existing constructor then we won't be able to take care of the last one, since we can't have nullable arguments to attribute constructor.

Reply:
Also, if we add a new argument to the existing constructor then that will become a breaking change as users will need to update code after they update the SDK.